azavea/pfb-network-connectivity
# Feature/kak/job upload details#648

## Overview

Display analysis import task status for its related analysis job in the web admin UI page list view. Also display additional analysis import task information on the detail page for its related analysis job.

Adds some import task fields to the related analysis job serializer. Also adds the ability to filter the import task endpoint directly by analysis job UUID, which is currently unused by the front-end site.


### Demo

Analysis job list view, with one job created via import task:
![image](https://user-images.githubusercontent.com/960264/51700951-4efe4c00-1fde-11e9-8632-976af3a0f94e.png)

The analysis job detail section for its import task:
![image](https://user-images.githubusercontent.com/960264/51700979-650c0c80-1fde-11e9-80c3-afd125aa2fe0.png)

Serialized analysis job response with related import task:
![image](https://user-images.githubusercontent.com/960264/51701018-7e14bd80-1fde-11e9-8cd9-6948d8c3c17c.png)


### Notes

I went with serializing a few extra fields on the analysis job rather than adding separate front-end queries for the related import task for each job, as doing so doesn't add much data to the existing analysis jobs query, should be more efficient network use than multiple additional queries, and more straightforward to implement than chained queries. Related batches, also shown in the analysis job list view and details, are also serialized directly with the analysis jobs.

## Testing Instructions

 * Create a new analysis import task, which should redirect on success to the analysis job list
 * Status for the task should show with its related analysis job in the list view
 * The related analysis job detail view should should the import task details as expected

Close #648
